static/app/views/dashboardsV2/widgetCard/metricsWidgetQueries.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+
+import {Client} from 'sentry/api';
+import {MetricsApiResponse, OrganizationSummary, PageFilters} from 'sentry/types';
+import {Series} from 'sentry/types/echarts';
+import {TableDataWithTitle} from 'sentry/utils/discover/discoverQuery';
+
+import {Widget} from '../types';
+
+type Props = {
+  api: Client;
+  children: (
+    props: Pick<State, 'loading' | 'timeseriesResults' | 'tableResults' | 'errorMessage'>
+  ) => React.ReactNode;
+  organization: OrganizationSummary;
+  selection: PageFilters;
+  widget: Widget;
+  limit?: number;
+};
+
+type State = {
+  errorMessage: undefined | string;
+  loading: boolean;
+  queryFetchID: symbol | undefined;
+  rawResults: undefined | MetricsApiResponse[];
+  tableResults: undefined | TableDataWithTitle[];
+  timeseriesResults: undefined | Series[];
+};
+
+class MetricsWidgetQueries extends React.Component<Props, State> {
```

Comment:
should we have a function instead of a class?

Reply:
I made it a class on purpose - to stay consistent with the `widgetQueries` and `issueWidgetQueries`.